cubes/query/browser.py

```diff
@@ -75,6 +77,41 @@
 _ReportResult = Union[AggregationResult, Facts, JSONType, List[JSONType]] 
 
 
+class BrowserFeatureAction(Enum):
+    aggregate = 1
+    fact = 2
+    facts = 3
+    members = 4
+    cell = 5
+
+    @classmethod
+    def from_string(cls, actions: Iterable[str]) -> List['BrowserFeatureAction']:
+        if not actions:
+            return []
+
+        return [cls[action] for action in actions if action in cls.__members__.keys()]
```

Comment:
That is ok. Through the list of `actions` (not sure if the best name name, should be rather `methods` or even better but more generic `capabilities`) the backend is telling us how much of the interface it has implemented or is capable of doing. For example, we might have a backend that can provide aggregate data but can't provide data at the fact/record level.